users/views.py

```diff
@@ -263,6 +151,11 @@ def home(request):
                 new_hours = request.POST.get('hours')
                 new_subject = request.POST.get('subject')
                 new_end = request.POST.get('end_time')
+                new_sched = ""
+                for s in sched:
+                    new_sched += s + " "
```

Comment:
You can compress this into one line: 
```python
new_sched = " ".join(sched)
```

Reply:
I'll fix the labelling issue after the feature is done overall

Reply:
Great, feel free to merge na after